includes/CategoryViewer.php

```diff
@@ -379,11 +377,9 @@ function getSubcategorySection() {
 			$r .= "<div id=\"mw-subcategories\">\n";
 			$r .= '<h2>' . wfMsg( 'subcategories' ) . "</h2>\n";
 			$r .= $countmsg;
-			/* Wikia change: $position param: top/bottom */
-			$r .= $this->getSectionPagingLinks( 'subcat', 'top' );
+			$r .= $this->getSectionPagingLinks( 'subcat' );
 			$r .= $this->formatList( $this->children, $this->children_start_char );
-			$r .= $this->getSectionPagingLinks( 'subcat', 'bottom' );
-			/* Wikia change end */
+			$r .= $this->getSectionPagingLinks( 'subcat' );
 			$r .= "\n</div>";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "\n</div>" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)